noxfile.py

```diff
@@ -33,7 +34,7 @@
 nox.options.error_on_missing_interpreters = False
 
 # Python versions to test against
-PYTHON_VERSIONS = ("3", "3.7", "3.8", "3.9", "3.10")
+PYTHON_VERSIONS = ("3", "3.9", "3.10")
```

Comment:
Given that python support is only from 3.10 `python_requires = >= 3.10`, do we need to test it against 3.9?

Reply:
Fixing that, missed a spot